samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -0,0 +1,163 @@
+.. _direction_finding_connectionless_tx:
+
+Bluetooth: Direction Finding Connectionless Beacon
+##################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+A simple application demonstrating the BLE Direction Finding CTE Broadcaster functionality
+by transmission of Constant Tone Extension with periodic advertising PDUs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix to use it in Angle of Departure mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix or any other antenna matrix.
+
+Configuration
+*************
+
+|config|
+
+The sample supports two Direction Finding modes:
+
+* Angle of Arrival
+* Angle of Departure
+
+By default both modes are available in the sample application.
+
+Angle of Arrival mode
+=====================
+
+To build this sample with Angle of Arrival mode only, set :makevar:`OVERLAY_CONFIG`
+to :file:`overlay-aoa.conf`.
+
+See :ref:`cmake_options` for instructions on how to add this option.
+For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars`
+in the Zephyr documentation.
+
+Antenna matrix configuration
+============================
+
+To use this sample with Angle of Departure enabled, additional GPIOS configuration
+is required to enable control of the antenna array. Example of such configuration
+is provided in :file:`nrf52833dk_nrf52833.overlay`.
+
+:file:`nrf52833dk_nrf52833.overlay` is a device tree overlay that provides information
+about which GPIOS should be used by Radio peripheral to switch between antenna
+patches during CTE transmission in AoD mode. At least two GPIOs must be provided
+to enable antenna switching. GPIOS are used by Radio peripheral in order following
+indices of :code:`dfegpio#-gpios` properties. The order is important because it
+affects mapping of antenna switch patterns to GPIOS (see `Antenna patterns`_).
+
+To successfully use Direction Finding Beacon with AoD mode enabled provide
+following data related with owned antenna matrix design:
+
+* provide GPIO pins to :code:`dfegpio#-gpios` propreties in :file:`nrf52833dk_nrf52833.overlay` file
+* provide default antenna that will be used to transmit PDU :code:`dfe-pdu-ant`
+  property in :file:`nrf52833dk_nrf52833.overlay` file
+* update antenna patterns in :cpp:var:`ant_patterns` array in :file:`main.c`.
```

Comment:
```suggestion
* Provide the GPIO pins to :code:`dfegpio#-gpios` properties in :file:`nrf52833dk_nrf52833.overlay` file
* Provide the default antenna that will be used to transmit the PDU :code:`dfe-pdu-ant` property in :file:`nrf52833dk_nrf52833.overlay` file
* Update the antenna switching patterns in :cpp:var:`ant_patterns` array in :file:`main.c`.
```
Not sure if :cpp:var: is correct.
There are following Breathe domains:
- Function: :c:func:
- Structure: :c:struct:
- Type: :c:type:
- Enum (the list): :c:enum:
- Enumerator (an item): :c:enumerator:
- Macro or define: :c:macro:
- Structure member: :c:member:

Any of these can be used?

Reply:
Corrected line breaks. Also I used :c:member: instead of :cpp:var:. Lets see outptus.

Reply:
Good, remember to apply other suggested changes.

Reply:
Added missing part, I'm sorry I've forgotten about it.

Reply:
What is wrong with this one?